### ISSUE TITLE
New Listings - A problem with track switches

### ISSUE THREAD

**Issue by birdofpreyru** [appirio-tech/topcoder-app#1161]
Steps to reproduce:
1. Go to the new listing page;
2. Switch off Design and Data Science track switches;
3. Reload the page (the state of switches will persists and you'll see dev challenges only);
4. Switch on the Design switch. No design challenges will appear until you reload the whole page!

**Comment by tladendo** [comment]
@birdofpreyru I can't reproduce so far. Are you still seeing this? Any other info?

**Comment by birdofpreyru** [comment]
@tladendo Yes, I still see the issue. Here is the video: https://youtu.be/6xxYF7g3pCU

**Comment by chq-matteo** [comment]
@tladendo @birdofpreyru is this issue going to get a bounty?

**Comment by chq-matteo** [comment]
Anyways I believe that I solved this issue while I was working on [Issue #1168](https://github.com/appirio-tech/topcoder-app/issues/1168)
